src/main/java/com/internet/shop/controllers/IndexController.java

```diff
@@ -5,11 +5,17 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.apache.log4j.Logger;
 
 public class IndexController extends HttpServlet {
+    static final Logger logger = Logger.getLogger(IndexController.class);
```

Comment:
let's make this a constant and add the access modifier

Reply:
done